src/components/policy/src/policy/src/cache_manager.cc

```diff
@@ -228,8 +228,13 @@ bool CacheManager::ApplyUpdate(const policy_table::Table& update_pt) {
       pt_->policy_table.app_policies_section.apps[iter->first].set_to_null();
       pt_->policy_table.app_policies_section.apps[iter->first].set_to_string("");
     } else if (policy::kDefaultId == (iter->second).get_string()) {
-      pt_->policy_table.app_policies_section.apps[iter->first] =
-          pt_->policy_table.app_policies_section.apps[kDefaultId];
+        policy_table::ApplicationPolicies::const_iterator iter_default =
```

Comment:
It's really hard to read - "the method of the value of the map of the field of the field of field".
Feel free to crate a local const (or not) reference and use it.